tests/groups/test_data.py

```diff
@@ -120,6 +122,20 @@ async def test_ok(
 
         assert await get_row_by_id(pg, SQLGroup, group.id) is None
 
+        async with (AsyncSession(pg) as session):
+            users = (
+                (
+                    await session.execute(
+                        select(user_group_associations).where(
+                            user_group_associations.c.group_id == group.id
+                        )
+                    )
+                )
+                .scalars()
+                .all()
+            )
+        assert len(users) == 0
```

Comment:
Won't this be true before and after deletion? with no associated users there should never be an entry in this table

Reply:
updated by adding user and group to user before deletion.